### ISSUE TITLE
Package triage; vendor documentation

### ISSUE THREAD

**Issue by mkevenaar** [chocolatey/choco-wiki#114]
[I want to take over/help with package maintenance for my software](https://chocolatey.org/docs/package-triage-process#i-want-to-take-overhelp-with-package-maintenance-for-my-software)

Based on the documentation, the site-admins should add the user to the package, no questions asked

I think the site-admins should verify that the sender is actually the vendor of the software.

**Comment by pauby** [comment]
Agreed. I don't think we should be too detailed on how it's verified (I just see that asking for problems later) but we need to make sure it's clear.
